README.md

```diff
@@ -1,22 +1,22 @@
 # The Developer Experience (DX) Automator
 
-This tool is intended to help make managing multiple Github repositories much easier for DX, DevRel, and Open Source Engineering teams. 
+This tool is intended to help make managing, the multiple Github repositories much easier for DX, DevRel, and Open Source Engineering teams. 
```

Comment:
Please do not edit this sentence. Thank you.

Reply:
Thank you for reviewing! Resolved the issue.